### PR TITLE
Update section title, close #196

### LINKED ISSUE #196: Change title of the section

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Seems like a no brainer so merging now FYI @mvl22. Quickfire reviews appreciated.

**Comment by mvl22** [comment]
See my comment.

**Comment by Robinlovelace** [comment]
> See my comment above that you missed.

Which comment? I think you need to submit the review before the comment is visible.

Note: I hit that issue, where I commented on a PR but it was invisible because I hadn't submitted the review, multiple times while I was getting started with reviewing PRs. May have missed something but there's no comment here as far as I can tell. Screenshot below:

image

**Comment by Robinlovelace** [comment]
A way to verify whether or not a comment is available to others: open the link in a private tab in which you're not linked in. Can you see it?

**Comment by mvl22** [comment]
IMG_4003

**Comment by mvl22** [comment]
> OK, so yes, seems that the review is still Pending. Can you click on the Files changed button in the top right and complete the review?

What an awful UI - the mixture of asynchronous commenting, which gives the appearance of being committed, and a submit button ('Review') is most unclear...

Now done anyway.

File: index.html [hunk 1]
```diff
@@ -241,9 +241,9 @@ <h2>Layer controls</h2>
 				<div id="cohesivenetworklegend" class="legend"></div>
 			</div>
 			
-			<button class="accordion">Level of Service (LoS) and existing infrastructure</button>
+			<button class="accordion">Cycling by design compliance</button>
```
[the remark this answers is not in the document]
Comment (Robinlovelace) [reply index.html:244]: Get it now, good point!

**Pull request merged**